File: code-coaches/easy/easter-eggs/easter_eggs.py
def main():
    total_eggs = int(input())
    eggs_found_by_me = int(input())
    eggs_found_by_friend = int(input())
    eggs_found = eggs_found_by_me + eggs_found_by_friend

    if eggs_found == total_eggs:
        print("Candy Time")
    else:
        print("Keep Hunting")


if __name__ == "__main__":
    main()

File: code-coaches/easy/easter-eggs/EasterEggs.java
import java.util.Scanner;

public class EasterEggs {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int totalEggs = input.nextInt();
		int eggsFoundByMe = input.nextInt();
		int eggsFoundByFriend = input.nextInt();
		int eggsFound = eggsFoundByMe + eggsFoundByFriend;

        if (eggsFound == totalEggs) {
	    	System.out.println("Candy Time");
        } else {
	    	System.out.println("Keep Hunting");
        }

		input.close();
    }

}
